dataformats/multigrid/src/MapFile.cpp

```diff
@@ -22,16 +23,20 @@ MapFile::MapFile(std::string filename) {
   }
 
   auto res = fstat(fd, &sb);
-  assert(res != -1);
```

Comment:
What's wrong with using assert() ? I think it is cleaner. Is it because 
it is compiled away wjen in Release mode?